### PR TITLE
DLP: Added sample for redact image with test cases

### PR DESCRIPTION
**Pull request by jainamtrivedi-crest**
- Added sample for [Redact Image](https://cloud.google.com/dlp/docs/samples/dlp-redact-image) with test cases.

### CODE REVIEW

**Review by annadata — APPROVED** [review]

**Comment by amanda-tarafa** [comment]
Please do not merge samples that have not been approved by @GoogleCloudPlatform/dotnet-samples-reviewers . I'll try to see if we can enforce it somehow.

File: dlp/api/Snippets/RedactImage.cs [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            InspectConfig = new InspectConfig
+            {
+                MinLikelihood = Likelihood.Likely,
+                IncludeQuote = true,
+                InfoTypes =
+                    {
+                        new InfoType { Name = "PHONE_NUMBER" },
+                        new InfoType { Name = "EMAIL_ADDRESS" },
+                        new InfoType { Name = "CREDIT_CARD_NUMBER" }
+                    }
```
Comment (amanda-tarafa) [inline dlp/api/Snippets/RedactImage.cs:39]: Wron indent.

**Review by amanda-tarafa — COMMENTED** [review]
Tiny style comment, thanks.

File: dlp/api/Snippets/RedactImage.cs (hunk 1 above)
Comment (jainamtrivedi-crest) [reply dlp/api/Snippets/RedactImage.cs:39]: I have raised a new PR: https://github.com/GoogleCloudPlatform/dotnet-docs-samples/pull/2355

**Pull request merged**